api/utils/ec2.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
```

Comment:
I'm not a fan of these utils packages where we dump a bunch of unrelated code. It contributes to binary bloat which we've spent a bit of time fixing.

Especially so here, when we already have an `api/utils/aws` package - can this EC2 stuff live in there?